openmrs-19/src/main/java/org/motechproject/openmrs19/util/ConverterUtils.java

```diff
@@ -429,7 +444,7 @@ public static Patient toPatient(OpenMRSPatient patient, OpenMRSPerson savedPerso
         Person person = new Person();
         person.setUuid(savedPerson.getPersonId());
         converted.setPerson(person);
-
+        converted.getPerson().getPreferredAddress().setPreferred("true");
```

Comment:
can you explain why you add this ?

btw. you can setPreferred before setting person in converted